src/main/java/com/cloudbees/jenkins/plugins/mtslavescloud/MansionCloud.java

```diff
@@ -473,4 +536,6 @@ public static boolean isInDevAtCloud() {
      * we continuously have problems provisioning or launching slaves.
      */
     public static Long MAX_BACKOFF_SECONDS = Long.getLong(MansionCloud.class.getName() + ".maxBackOffSeconds", 600);  // 5 minutes
+
+    public long lastLaunchedSlaveTimeInEpoch = System.currentTimeMillis()/1000;
```

Comment:
non private fields are an anti-pattern, favour using a getter/setter for access outside of this class (e.g. if you want to access this via the groovy console for debugging) otherwise make private... also fields that are not static should be at the top of the class (in my opinion all fields should be at the top of the class, but the code convention here is static fields at the bottom)